docs/src/examples/components/Avatar/Usage/AvatarUsageExample.shorthand.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+import { Avatar } from '@stardust-ui/react'
+
+const status = { color: 'green', icon: 'check', title: 'Available' }
+
+const AvatarUsageExampleShorthand = () => (
```

Comment:
just a note for future: for all the components we have this section where examples that are rather theme-specific are provided. It is a clear indicator that we should think about mechanisms for theme developers to define custom docs section where theme-specific examples will be provided.